e2e/test.js

```diff
@@ -78,10 +78,10 @@ test('Sample Game', async t=> {
                 gamePage.cardsInHand.nth(0),
                 gamePage.rows.nth(0))
             .expect(gamePage.cardsInHand.count).eql(num-1)
-            .expect(gamePage.rows.nth(0).find('.card').count).eql(cardsInPlay+2)
+            .expect(gamePage.rows.nth(0).find('.card').count).eql(cardsInPlay+1)
```

Comment:
Aren't we now missing a test to see the opponent's row has the expected number of cards?

Reply:
True, also, I just realized that it CAN be unit-tested 🤔

Reply:
gamePage.rows.nth(0) should be declared outside the loop (it appears too many times), the rest could also recive the same treatment, but it's way less dramatic, and num variable name could also be renamed to maxHandNum or something like that.

After the second dragToElement i would add a
`
.then(() => cardsInPlay+=1)
`
this way the variable cardsInPlay is actually storing the number of cards in play

Reply:
putting that `.then` in would imply needing a second `await t`... and I don't think it woud be either more readably or more correct, it's just something we want to update every loop.

Agreed on the rest.